packages/frontend/src/hooks/useArbitratorsDisputeFee.ts

```diff
@@ -30,11 +30,11 @@ export function useArbitratorsDisputeFee(
         if (!chain) return;
         if (!fees) return;
         setArbitratorsDisputeFee(
-            (fees as BigNumber[]).reduce((accumulator, current, index) => {
+            fees.reduce((accumulator, current, index) => {
                 return {
                     ...(accumulator as object),
-                    [addresses[index]]: utils.formatUnits(
-                        current,
+                    [addresses[index]]: formatUnits(
+                        current.result as bigint,
```

Comment:
Here too

Reply:
`result` is `unknown` here, couldn't figure out how to do it with the typings.

Reply:
Yeah let's leave it like this. I had these issues on the ERC20 KPI token template too. `wagmi` should infer the types correctly but it doesn't.